packages/evm-adapter/.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
you won't need this config if you use our prettier plugin similar to solana adapter

Reply:
```JSON
"eslintConfig": {
    "plugins": [
      "prettier",
      "jest"
    ],
    "rules": {
      "prettier/prettier": "error"
    },
    "extends": [
      "@heliofi/eslint-config-helio",
      "prettier"
    ],
    "parserOptions": {
      "project": "./tsconfig.json"
    }
  },
  "prettier": {
    "singleQuote": true
  },
  ```

Reply:
Ok